package.json

```diff
@@ -4,7 +4,8 @@
   "private": true,
   "description": "The NI Nimble Design System Monorepo",
   "scripts": {
-    "postinstall": "patch-package",
+    "postinstall": "patch-package && npm run playwright:setup",
+    "playwright:setup": "playwright install --with-deps",
```

Comment:
The playwright version should be captured in the devDependencies (this package.json is relying on that package to be installed but didn't declare a dependency on it, very fishy)